common/Helpers/EnumToIntConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Data;
+
+namespace DevHome.Common.Helpers;
+
+public class EnumToIntConverter : IValueConverter
+{
+    public EnumToIntConverter()

```

Comment:
Why the zero argument Constructor that does nothing?

Reply:
The converter I converted (ha) this from also had it.

Reply:
File deleted.